master/internal/api_tensorboard.go

```diff
@@ -69,24 +70,22 @@ func (a *apiServer) LaunchTensorboard(
 		trialIds = append(trialIds, int(id))
 	}
 
-	cmdParams, user, err := a.prepareLaunchParams(ctx, &protoCommandParams{
+	params, err := a.prepareLaunchParams(ctx, &protoCommandParams{
 		TemplateName: req.TemplateName,
 		Config:       req.Config,
 		Files:        req.Files,
+		MustZeroSlot: true,
 	})
 	if err != nil {
 		return nil, err
 	}
 
 	tensorboardConfig := command.TensorboardRequest{
-		CommandParams: cmdParams,
+		CommandParams: params,
 		ExperimentIDs: experimentIds,
 		TrialIDs:      trialIds,
 	}
-	tensorboardLaunchReq := command.TensorboardRequestWithUser{
-		Tensorboard: tensorboardConfig,
-		User:        user,
```

Comment:
question: is the user portion of this type used anywhere else? it seems we can get rid of it.

Reply:
It is not gotten rid of. It is merged into the `Tensorboard` field. I just reused the same go struct.

Reply:
aah I didn't explain it well I meant something like this https://github.com/determined-ai/determined/pull/2377 how does this look?

Reply:
It looks good! I approved it